src/ConfigManager.cpp

```diff
@@ -1,7 +1,8 @@
 #include "ConfigManager.h"
 
 const byte DNS_PORT = 53;
-const char magicBytes[2] = {'C', 'M'};
+const char magicBytesSet[2] = {'C', 'M'};
```

Comment:
`magicBytesSet` seems weird to me, I would leave it as `magicBytes`.

Reply:
`magicBytes` and `magicBytesEmpty` ?

Reply:
I would say.